lib/tx_build/set_trustline_flags.ex

```diff
@@ -0,0 +1,79 @@
+defmodule Stellar.TxBuild.SetTrustlineFlags do
+  @moduledoc """
+  Allows an issuing account to configure various authorization
+  and trustline flags for all trustlines to an asset.
+  """
+  import Stellar.TxBuild.Validations,
+    only: [
+      validate_account_id: 1,
+      validate_asset: 1,
+      validate_trustline_flags: 1,
```

Comment:
@einerzg since this validation only concerns this operation, let's place it as a private function. The main reason behind this is to keep just common validations in `Stellar.TxBuild.Validations`. What do you think?